### PR TITLE
8/30/2024 Production Deploy

### PR DESCRIPTION
**Pull request by ccostino**
*A note to PR reviewers: it may be helpful to review our [code review documentation](https://github.com/GSA/notifications-api/blob/main/docs/all.md#code-reviews) to know what to keep in mind while reviewing pull requests.*

## Description

This is a production deploy for 8/30/2024.  [Full release notes are here!](https://github.com/GSA/notifications-api/releases/tag/08-30-2024)

## Security Considerations

* All changes have been reviewed and tested prior to being included here.

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ class Config(object):
     SQLALCHEMY_POOL_TIMEOUT = 30
     SQLALCHEMY_POOL_RECYCLE = 300
     SQLALCHEMY_STATEMENT_TIMEOUT = 1200
-    PAGE_SIZE = 50
+    PAGE_SIZE = 20
```
Comment (terrazoon) [inline app/config.py:87]: does this have an associated admin-side PR?

Comment (ccostino) [reply app/config.py:87]: I was wondering about this too - I think it does, https://github.com/GSA/notifications-admin/pull/1882

**Review by terrazoon — APPROVED** [review]
Approved assuming Bev and Cliff think it's safe to push API only.

**Review by A-Shumway42 — APPROVED** [review]
LGTM

File: app/config.py (hunk 1 above)
Comment (heyitsmebev) [reply app/config.py:87]: Yes, related to the pagination on UI side

**Pull request merged**